packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -108,19 +108,55 @@ <h2 id="network" data-translate="pci_database_add_step5_title"></h2>
                         data-value="false"
                         data-disabled="$ctrl.loadingSubnets || !$ctrl.model.flavor.supportsPublicNetwork"
                     >
-                        <span
-                            data-translate="pci_database_add_network_mode_public"
-                        ></span>
+                        <div class="d-flex align-items-center">
+                            <span
+                                data-translate="pci_database_add_network_mode_public"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                data-oui-popover="{{'pci_database_add_network_mode_public_popover' | translate}}"
+                                data-oui-popover-placement="right"
+                            ></button>
+                        </div>
                     </oui-radio>
                     <oui-radio
                         data-value="true"
                         data-disabled="$ctrl.loadingSubnets || !$ctrl.model.flavor.supportsPrivateNetwork"
                     >
-                        <span
-                            data-translate="pci_database_add_network_mode_private"
-                        ></span>
+                        <div class="d-flex align-items-center">
+                            <span
+                                data-translate="pci_database_add_network_mode_private"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                data-oui-popover="{{'pci_database_add_network_mode_private_popover' | translate}}"
+                                data-oui-popover-placement="right"
+                            ></button>
+                        </div>
                     </oui-radio>
                 </oui-radio-group>
+                <div class="oui-field__helper">
+                    <span
+                        data-translate="pci_database_add_network_mode_helper"
+                        data-translate-values="{
+                        link: 
+                        '<a 
+                            class=&quot oui-link_icon &quot 
+                            href=&quot' + $ctrl.privateNetworkGuideUrl + '&quot 
+                            target=&quot _blank &quot 
+                        >
+                            <span>{{:: 'pci_database_add_network_mode_helper_link' | translate}}</span>
+                            <span 
+                                class=&quot oui-icon oui-icon-external-link &quot 
+                                aria-hidden=&quot true &quot 
+                            ></span>
+                        </a>',
+                    }"
+                    >
+                    </span>
```

Comment:
**question:** Could you please try removing `&quot` and placing `data-translate-compile directive on the `span`? I am not sure if it works, but its worth a try.
```suggestion
                    <span
                        data-translate="pci_database_add_network_mode_helper"
                        data-translate-values="{
                        link: 
                        '<a 
                            class="oui-link_icon" 
                            href="t' + $ctrl.privateNetworkGuideUrl + '"
                            target="_blank"
                        >
                            <span data-translate="pci_database_add_network_mode_helper_link"></span>
                            <span 
                                class="oui-icon oui-icon-external-link" 
                                aria-hidden="true"
                            ></span>
                        </a>',
                    }"
                    data-translate-compile
                    >
                    </span>
```

Reply:
Hi @anooparveti ,
We already had a lot of discussion on this point, I tried many suggestions, none of them seems to work, this one included:
- https://github.com/ovh/manager/pull/7460#discussion_r910689270
- https://github.com/ovh/manager/pull/7460#discussion_r927484917
- https://github.com/ovh/manager/pull/7460#discussion_r935539536

Replacing `&quot` with `"`  creates invalid html, which obviously does not work:
![image](https://user-images.githubusercontent.com/24541481/184089445-dc51c125-6c69-43b5-91f1-cb75100bc134.png)
![image](https://user-images.githubusercontent.com/24541481/184090424-0d8cc1fb-6476-46d7-9366-4236deb9963e.png)


I understand that we should try to avoid splitting sentences and having html in the translation files, but it seems to me that we don't have a workaround working for every case.

Here, the code with `&quot` is working, but it's still not perfect as we have to use `&quot` , and it is quite hard to read.